src/fluree/db/query/exec.cljc

```diff
@@ -80,7 +80,9 @@
   [subquery]
   (fn [ds fuel-tracker error-ch]
     (->> (execute* ds fuel-tracker subquery error-ch)
-         (select/subquery-format ds subquery fuel-tracker error-ch))))
+         (select/subquery-format ds subquery fuel-tracker error-ch)
+         (drop-offset subquery)
```

Comment:
Tiny suggestion that you can take or leave, but it might be better to group these `drop-offset` and `take-limit`  into a `page` function since they will always be called together and need to be called in two different places at least.